x-pack/plugins/search_indices/server/lib/status.test.ts

```diff
@@ -116,6 +117,7 @@ describe('status api lib', function () {
   });
 
   describe('fetchUserStartPrivileges', function () {
+    const testIndexName = generateRandomIndexName();
```

Comment:
We can just mock a random index name here

```suggestion
    const testIndexName = 'test-index-0a230';
```

Reply:
added in https://github.com/elastic/kibana/pull/197869/commits/21fbf903e93e029df109021ff874e83164c550c2